### ISSUE TITLE
Can I study the machine learning model code built using AttBLSTM?

### ISSUE THREAD

**Issue by leesk212** [stefan1wan/InvisibleProbe#3]
Hello, I am Seok-Min Lee, a master's student studying machine learning at Korea University in Korea.

My biggest concern lately is using perf or PCM to detect attack states*. (* Attack status: CPU-related Meltdown, Specter attack, or PCI-E-related Status attack)

So, when I read the Invisible Probe paper, I was very shocked by the attack using PCI-E Congestion for the shared resources of the attacker and the victim.

Furthermore, through the corresponding counter values, Website Fingerprinting. It was really impressive to proceed with other ML attacks.

In conclusion, I would like to learn about machine learning models you or your team developed purely for research purposes. In particular, I want to learn about the dataset you used and the model code.

I'm sorry for the sudden contact. I'm very curious.

If you don't mind, I would like to receive the machine learning-related code and dataset used in the paper.

I am looking forward to hearing from you.

Best Regards

**Comment by stefan1wan** [comment]
Sorry for responding late. The AI part was finished by my colleague, I have already had him know. I think he will give you a link later~
